src/ReactiveSocket.h

```diff
@@ -51,45 +51,47 @@ class ReactiveSocket {
       ConnectionSetupPayload setupPayload = ConnectionSetupPayload(),
       Stats& stats = Stats::noop(),
       std::unique_ptr<KeepaliveTimer> keepaliveTimer =
-          std::unique_ptr<KeepaliveTimer>(nullptr));
+          std::unique_ptr<KeepaliveTimer>(nullptr),
+      folly::Executor& executor = defaultExecutor());
```

Comment:
I don't think this argument should be defaulted. In our mobile code, won't we always want to use Liger's `EventBase` here? It seems like something that the caller should always carefully consider.